### PR TITLE
D.sulym/homework/04 classes

### PR DESCRIPTION
**Pull request by DMSamael**

### CODE REVIEW

File: ConsoleApp1/ConsoleApp1/Program.cs [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+{
+    public class Homework
+    {
+        static void Main(string[] args)
+        {
+            
+        }
+
+        //створити метод Max приймає від 2 до 5 аргументів та повертає max відповідно значення серед цих аргументів
+        public static int MaxV(int? v1, int? v2, int? v3 = null, int? v4 = null, int? v5 = null)
```
Comment (misikdmitriy) [inline ConsoleApp1/ConsoleApp1/Program.cs:13]: it's ok, but do not use nullable until we're going to learn it

File: ConsoleApp1/ConsoleApp1/Program.cs [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            return result;
+        }
+
+        //створити метод TryMulIfDividedByThree що повертає булеве значення true/false якщо хоче б одне з чисел ділиться на 3. за допомогою out параметру повернути результат добутку(або нуль якщо обидва числа не діляться на 3)
+
+        public static bool TryMulIfDividedByThree(double var1, double var2,out double result)
+        {
+            if(var1%3==0 || var2 % 3 == 0)
+            {
+                result = var1 / var2;
```
Comment (misikdmitriy) [inline ConsoleApp1/ConsoleApp1/Program.cs:64]: multiply, not divide :)

File: ConsoleApp1/ConsoleApp1/Program.cs [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+            }
+            return concat;
+        }
+        //створити функцію Pow(x, y) що буде підносити X в степінь Y(x та y цілі невід'ємні числа)
+        public static int Pow(int value, int power)
+        {
+            int result = 0;
+            if (power > 0 && power != 1)
+            {
+                result = value * Pow(value, power - 1);
```
Comment (misikdmitriy) [inline ConsoleApp1/ConsoleApp1/Program.cs:90]: not critical, you can return right here
return value * Pow(value, power - 1);

File: ConsoleApp1/ConsoleApp1/Program.cs [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+            }
+            else
+            {
+                result = value;
```
Comment (misikdmitriy) [inline ConsoleApp1/ConsoleApp1/Program.cs:94]: and can return here 
return value

**Review by misikdmitriy — CHANGES_REQUESTED** [review]
please update your homework to use only features we've learnt before like conditions/loops and methods

File: ConsoleApp1/ConsoleApp1/Program.cs (hunk 1 above)
Comment (DMSamael) [reply ConsoleApp1/ConsoleApp1/Program.cs]: Fixed

File: ConsoleApp1/ConsoleApp1/Program.cs (hunk 2 above)
Comment (DMSamael) [reply ConsoleApp1/ConsoleApp1/Program.cs]: Fixed

File: ConsoleApp1/ConsoleApp1/Program.cs (hunk 3 above)
Comment (DMSamael) [reply ConsoleApp1/ConsoleApp1/Program.cs:67]: I think it can be too many returns in big structure, so such code much easier to read. No?

**Pull request closed without merging**